.github/workflows/ci.yml

```diff
@@ -12,31 +12,8 @@ on:
     branches: [main]
 
 jobs:
-
```

Comment:
change to this linter instead of removing linting?

Reply:
Linter is now in a separate github action yaml. We still have the linter in `yapf.yml` and `pylint.yml`. Do you think I need to move this part back into `ci.yml`?

Reply:
can you merge yapf and pylint into one job in ci.yml and let test depend on it (like I did). Otherwise, test will be trigger even if linting is not passed, which wates CI resources

Reply:
OK sounds good.